resources/views/partials/example-requests/python.blade.php

```diff
@@ -0,0 +1,27 @@
+```python
+import requests
+import json
+
+url = '{{ trim(config('app.docs_url') ?: config('app.url'), '/')}}/{{ ltrim($route['uri'], '/') }}'
+@if(count($route['bodyParameters']))
+payload = {
+	@foreach($route['bodyParameters'] as $attribute => $parameter)
+	    '{{ $attribute }}': '{{ $parameter['value'] }}'@if(!($loop->last)),@endif  {{ !$parameter['required'] ? '# optional' : '' }}
+	@endforeach
+}
+@endif
+@if(count($route['queryParameters']))
+params = {
+	@foreach($route['queryParameters'] as $attribute => $parameter)
```

Comment:
https://laravel-apidoc-generator.readthedocs.io/en/latest/generating-documentation.html#specifying-language-for-examples -> cleanQueryParameters